HCL.ArticleService.API/Controllers/ArticleController.cs

```diff
@@ -83,7 +83,7 @@ public async Task<IResult> GetArticle(string filterOperator,string field, string
             return Results.NoContent();
         }
 
-        [HttpGet("GetArticle/{filterOperator}/{field}/{value}/{skipCount}/{takeCount}")]
+        [HttpGet("v1/GetArticle/{filterOperator}/{field}/{value}/{skipCount}/{takeCount}")]
         public async Task<IResult> GetArticles(string filterOperator, string field, string value,int skipCount,int takeCount)
```

Comment:
In route you shouldn't write GetArticle. And this params. Add [FromQuery] in method params